lib/grammar2.pegjs

```diff
@@ -72,7 +71,13 @@ ctrl = ifctrl
      / forctrl
      / returnctrl
 
-ifctrl     = "if" _ c:innerstmt _ b:block { return p.parseIf(c, b) }
+ifctrl = "if" _ c:innerstmt __ b:block ei:(__ elseifcont)* {
+  ei = ei.map(function (pair) { return pair[1] })
```

Comment:
Why are you transforming the set of ei expression arrays into the second element of the pair?